github-sample/.github/workflows/logicapp_deploy.yml

```diff
@@ -4,67 +4,82 @@
 name: (classic) Build and Deploy Logic App. 
 
 env:
-  AZURE_LOGICAPP_PACKAGE_PATH: '.'
-  DOTNET_VERSION: '3.1.301'    # set this to the dotnet version to use
-  RG_LA: 'your-la-rg'
-  RG_CON: 'your-con-rg'
-  LA_NAME: 'your-la-name'
+  LA_NAME: 'la-name-324234'
 
 on:
-  - workflow_dispatch 
+  workflow_dispatch:
+    # Trigger the workflow every time the build workflow ran to completion
+  workflow_run:
+    workflows:
+      - (classic) Logic App Infrastructure as Code Pipeline.
+    types:
+      - completed
+  # Triggers when an environment file has been changed  
+  push:
+    paths:
+      - "logic/**"
+      - ".github/workflows/logicapp_deploy.yml"
 
 jobs:
-   logic_app_build: 
+  logic_app_build: 
+    name: Build 
     runs-on: ubuntu-latest
     steps: 
-
     - name: Checkout
       uses: actions/checkout@v1
-    
+    - name: Package logic
+      run: (cd logic; mkdir -p ../build-art; zip -r ../build-art/logic.zip .)
```

Comment:
With this change one don't need to specify name of every workflow in this action like before? That's great.

Reply:
Should `build-art` be called `output` or something like that?